common/src/main/java/gov/cms/ab2d/common/model/Attestation.java

```diff
@@ -20,7 +23,8 @@
 public class Attestation {
 
     @Id
-    @GeneratedValue
+    @GeneratedValue(strategy = SEQUENCE, generator = "attestation_id_seq")
```

Comment:
I would like to reduce the clutter introduced by having to specify annotation attributes (strategy, generator, etc). Can't we just declare `@GeneratedValue` and rely on the defaults?

Reply:
With only `@GeneratedValue` I was getting and error when attempting to run the boot App. (details in the comment below)

Reply:
Changed to use the default `hibernate_sequence` are advised.